utils/firestore.js

```diff
@@ -791,6 +791,7 @@ const filterDB = async (queries, siteCode, isParent) => {
             if(key === 'connectId') query = query.where('Connect_ID', '==', parseInt(queries[key]));
             if(key === 'token') query = query.where('token', '==', queries[key]);
             if(key === 'studyId') query = query.where('state.studyId', '==', queries[key]);
+            if(key === 'checkedIn') query = query.where('331584571.266600170.135591601', '==', 353358909);
```

Comment:
Should we define conceptIDs in `fieldToConceptIdMapping.js`?

Reply:
not required since this is part of the query.

Reply:
Select visit from dropdown menu -> baseline -> Check-in complete == yes

looks good

Reply:
Variables for concept IDs will make the code more readable, for example:
```js
if (key === 'checkedIn') query = query.where(`{cid.dropdown}.{cid.baseline}.{cid.isCheckinComplete}`, '==', cid.yes);
```